Monika After Story/game/script-topics.rpy

```diff
@@ -11929,11 +11936,22 @@ label monika_player_appearance:
 
                 "My eyes are another color.":
                     $ persistent._mas_pm_eye_color = ask_color("What color are your eyes?")
+                    $ persistent._mas_pm_has_heterochromia = False
 
                     m 3hub "Oh! That's a beautiful color, [player]!"
                     m 2eub "I'm sure I could get lost for hours, staring into your [persistent._mas_pm_eye_color] eyes."
                     m 3hua "Now, onto my next question--"
 
+                "I have heterochromia.":
+                    $ persistent._mas_pm_has_heterochromia = True
```

Comment:
I'm not sure how useful this whole thing is, but if we do it, can as well ask eyes colours.

Reply:
Right, I think we can.